extension/src/popup/components/manageAssets/SearchAsset/index.tsx

```diff
@@ -137,6 +137,10 @@ export const SearchAsset = ({ setErrorAsset }: SearchAssetProps) => {
     setHasNoResults(!assetRows.length);
   }, [assetRows]);
 
+  if (isCustomNetwork(networkDetails)) {
+    return <Redirect to={ROUTES.addAsset} />;
+  }
```

Comment:
redirect to the manual "Add Asset" flow as the search will not help someone on a Custom network